packages/playwright/src/matchers/expect.ts

```diff
@@ -345,11 +345,7 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
 
       try {
         const callback = () => matcher.call(target, ...args);
-        // toPass and poll matchers can contain other steps, expects and API calls,
-        // so they behave like a retriable step.
-        const result = (matcherName === 'toPass' || this._info.poll) ?
-          zones.run('stepZone', step, callback) :
-          zones.run<ExpectZone, any>('expectZone', { title, stepId: step.stepId }, callback);
+        const result = zones.run('stepZone', step, callback);
```

Comment:
expectZone used to name the underlying _expect protocol call, do I get it right that ever since we started merging the traces it is no longer required?

Reply:
This now happens through `ClientSideInstrumentation` in the `index.ts` file.